src/test/java/com/exasol/adapter/dialects/oracle/OracleVirtualSchemaIntegrationTestSetup.java

```diff
@@ -69,25 +71,23 @@ public class OracleVirtualSchemaIntegrationTestSetup implements Closeable {
         }
     }
 
-    public static void uploadOracleJDBCDriverAndVSToBucket(final Bucket bucket)
+    public static void uploadOracleJDBCDriverToBucket(final ExasolContainer<? extends ExasolContainer<?>> container)
             throws BucketAccessException, TimeoutException, FileNotFoundException {
-        final String driverName = getPropertyFromFile(RESOURCES_FOLDER_DIALECT_NAME, "driver.name");
-
-        final Path pathToSettingsFile = Path.of("src", "test", "resources", "integration", "driver",
-                RESOURCES_FOLDER_DIALECT_NAME, JDBC_DRIVER_CONFIGURATION_FILE_NAME);
+        container.getDriverManager()
+                .install(JdbcDriver.builder("ORACLE").enableSecurityManager(false).mainClass("oracle.jdbc.OracleDriver")
+                        .prefix("jdbc:oracle:thin:")
+                        .sourceFile(Path.of("target/oracle-driver", ORACLE_JDBC_DRIVER_NAME)).build());
```

Comment:
Wow! I did not know this API!
In other VS I think we uploaded a a settings file, as in VSORA before. 
:thinking: